addons/stock/models/stock_move.py

```diff
@@ -130,6 +130,9 @@ def _default_group_id(self):
     picking_type_id = fields.Many2one('stock.picking.type', 'Operation Type')
     inventory_id = fields.Many2one('stock.inventory', 'Inventory')
     move_line_ids = fields.One2many('stock.move.line', 'move_id')
+    move_line_default_uom_id = fields.Many2one('product.uom', compute='_compute_move_line_default_uom',
```

Comment:
@sle-odoo Are you ok with this? Used in order to be able to add move line from a move with UOM and serial